jest.config.js

```diff
@@ -6,6 +6,10 @@
  */
 const path = require('path');
 
+const worker = require('worker_threads');
```

Comment:
This breaks `yarn test:dev`, which I use regularly. Please add the experimental flag!

Reply:
The experimental flag is present. Check out the package.json file.

Also tests pass in my local. Can you check if you are running an older verison?

Reply:
Done

![image](https://user-images.githubusercontent.com/35203638/68899419-3a1eb080-06f7-11ea-828f-cb7d62d619dd.png)